packages/webapp/src/App/Layout/Layout.tsx

```diff
@@ -24,6 +24,7 @@ import SdsManagerWorker from '~workers/SdsManager?worker'
 import type { ISynthesisInfo } from '~workers/types'
 import { createWorkerProxy } from '~workers/utils'
 
+import { PolicyAndCookieBanner } from '../../components/PolicyAndCookieBanner/PolicyAndCookieBanner.js'
```

Comment:
Perhaps this could be replaced by `import { PolicyAndCookieBanner } from '~components/PolicyAndCookieBanner'`?

Reply:
Done!